docs/manual-EN/1.overview/2.quick-start/4.import-csv-file.md

```diff
@@ -0,0 +1,243 @@
+# Example of Importing CSV Files
+
+The following example shows you how to import CSV data to **Nebula Graph** with **Nebula Importer**. In this example, **Nebula Graph** is installed with `Docker` and `Docker Compose`. We will walk you through the example by the following steps:
+
+1. [Start your Nebula Graph services](#starting-nebula-graph-services)
+2. [Create the schema for vertices and edges](#creating-the-schema-for-vertices-and-edges)
+3. [Prepare the configuration file](#preparing-your-configuration-file)
+4. [Prepare the CSV data](#preparing-the-csv-data)
+5. [Import the CSV data](#importing-the-csv-data)
+
+## Starting Nebula Graph Services
+
+You can start your **Nebula Graph** services by the following steps:
+
+1. On a command line interface, go to the `nebula-docker-compose` directory.
+2. Execute the following command to start **Nebula Graph** services:
+
+```bash
+$ sudo docker-compose up -d
+```
+
+3. Execute the following command to pull the **Nebula Graph** image:
+
+```bash
+$ sudo docker pull vesoft/nebula-console:nightly
+```
+
+4. Execute the following command to connect to your **Nebula Graph** server:
+
+```bash
+$ sudo docker run --rm -ti --network=host vesoft/nebula-console:nightly --addr=127.0.0.1 --port=3699
+```
+
+**Note**: You must ensure your IP address and port number are configured correctly.
+
+## Creating the Schema for Vertices and Edges
+
+Before you can input your schema, you must create a space and use it. In this example we create a **nba** space and use it. We create two tags and two edge types with the following commands:
+
+```ngql
+CREATE TAG player (name string, age int);
```

Comment:
nebula> CREATE TAG player (name string, age int);
Please add `nebula> `if it's a query.